x-pack/plugins/security_solution/scripts/endpoint/common/stack_services.ts

```diff
@@ -308,22 +309,13 @@ export const fetchKibanaStatus = async (kbnClient: KbnClient): Promise<StatusRes
  * Checks to ensure Kibana is up and running
  * @param kbnUrl
  */
-export const waitForKibana = async (kbnUrl: string): Promise<void> => {
-  const url = (() => {
-    const u = new URL(kbnUrl);
-    // This API seems to be available even if user is not authenticated
-    u.pathname = '/api/status';
-    return u.toString();
-  })();
-
+export const waitForKibana = async (kbnClient: KbnClient): Promise<void> => {
```

Comment:
can you edit the `@param` above in the jsdocs to match the new param name

Reply:
FYI: I seem to being able to use `kbnClient` for some reason when I initially added this method, but don't recall what that was now. Did you test this code here to ensure it works? (ex. ensuring that the `pRetry()` below is executes the callback at least twice)

Reply:
yes, I did, the previous solution was failing due to the `nodeFetch` missing cert for SSL endpoints